docs/setup/running_natively.md

```diff
@@ -19,6 +19,12 @@ they would be when running rails directly.
 - `bundle exec rake spec` - Run the entire test suite  ( for `rspec spec`). Test coverage statistics are in `coverage/index.html` or in [CodeClimate](https://codeclimate.com/github/department-of-veterans-affairs/vets-api/code)
 - `make guard` - Run the guard test server that reruns your tests after files are saved. Useful for TDD!
 
+### Running tests in parallel
+- NOTE: Running specs in parallel requires that your development database exists, and is up to date. If necessary, you may need to run `bundle exec rake db:create` and `bundle exec rake db:migrate` before the following steps
+- `RAILS_ENV=test bundle exec rake db:reset` - Set up your test database properly first. This just makes sure if your existing test database is named differently than what is expected, it will be resolved first
```

Comment:
I don't think `rake db:reset` is necessary now that you've reverted the change in settings.yml

Reply:
Sounds good, I'll make this change to remove the line